packages/deployment/ansible/roles/cosmos-genesis/files/faucet-helper.sh

```diff
@@ -5,7 +5,7 @@ thisdir=$(dirname -- "$0")
 FAUCET_HOME=$thisdir/../faucet
 
 MAX_LINES=-1
-DELEGATE_COINS=62000000ubld,93000000urun
+DELEGATE_COINS=62000000ubld,9300000urun
```

Comment:
Is dividing the µrun by ten the intent here?

Reply:
Yes, giving out $9.30 is more affordable than $93.00.